api/src/main/java/org/openmrs/api/db/ConditionDAO.java

```diff
@@ -53,6 +53,12 @@
 	 */
 	List<Condition> getActiveConditions(Patient patient);
 
+	/**
+	 * @see ConditionService#getAllConditions(Patient)
+	 * @since 2.4.0
```

Comment:
The since annotation should be only on the service interface, ConditionService

Reply:
@dkayiwa  I updated the PR. Please take a look :)